packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts

```diff
@@ -35,7 +47,10 @@ export class IncomingRequestsController extends ConsumptionBaseController {
         private readonly processorRegistry: RequestItemProcessorRegistry,
         parent: ConsumptionController,
         private readonly eventBus: EventBus,
-        private readonly identity: { address: CoreAddress }
+        private readonly identity: { address: CoreAddress },
+        private readonly relationshipResolver: {
+            getRelationshipToIdentity(id: CoreAddress): Promise<Relationship | undefined>;
```

Comment:
If you only want an active relationship, please rename this to getActiveRelationshipToIdentity

Reply:
no relationship would also be an acceptable result

Reply:
yeah - but you want active or none. Not just any one.

Reply:
getActiveRelationship would not distinguish between no relationship and inactive relationship

Reply:
It does not have to, we don't care. Actually you don't distinguish about that below.

Reply:
ok, I'll also think about this again!